Source/Extensions/Blazorise.Components/SelectList.razor.cs

```diff
@@ -22,14 +23,16 @@ public partial class SelectList<TItem, TValue> : ComponentBase
         /// </summary>
         private Select<TValue> selectRef;
 
+        private TValue selectedValue;
+
         #endregion
 
         #region Methods
 
         protected Task HandleSelectedValueChanged( TValue value )
         {
             SelectedValue = value;
-            return SelectedValueChanged.InvokeAsync( value );
```

Comment:
Any reason why this is removed?

Reply:
As changes also can be trigger from SelectedItem change, the raising of the events shall when the property changes.